pulumi/README.md

```diff
@@ -0,0 +1,82 @@
+# Pulumi
+
+Author: [Nick Santos](https://github.com/nicks)
+
+Use [Pulumi](https://www.pulumi.com/) to deploy services to a cluster.
+
+Use this to add a fast dev loop to your Pulumi-based projects.
+
+## Requirements:
+
+- Tilt v0.23.4+
+- Pulumi
+- Python3
+- MacOS, Linux, or WSL
+
+## Functions
+
+### pulumi_resource
+
+```
+
+def pulumi_resource(
+    name,
+    stack='',
+    dir='',
+    deps=[],
+    image_deps=None,
+    image_configs=None,
+    image_selector='',
+    container_selector='',
+    live_update=None,
+    resource_deps=None,
+    labels=None,
+    port_forwards=[]):
+  ...
+```
+
+Installs a pulumi stack to a cluster.
+
+Arguments:
+
+`name`: The name of the resource in the Tilt UI.
+
+`stack`: The stack to operate on. Defaults to the current stack.
+
+`dir`: The directory to run 'pulumi up' in. Defaults to the current working directory.
```

Comment:
is this relative or absolute?

Reply:
most tilt paths work the same way (unless otherwise noted) - relative to the directory of the currently executing tiltfile.